canal/canal.go

```diff
@@ -37,13 +38,17 @@ type Canal struct {
 	connLock sync.Mutex
 	conn     *client.Conn
 
-	tableLock sync.RWMutex
-	tables    map[string]*schema.Table
+	tableLock          sync.RWMutex
+	tables             map[string]*schema.Table
+	errorTablesGetTime map[string]time.Time
 
 	ctx    context.Context
 	cancel context.CancelFunc
 }
 
+// canal will retry fetching unknown table's meta after UnknownTableRetryIntervalSec
+var UnknownTableRetryIntervalSec = time.Second * time.Duration(10)
```

Comment:
This var name is a little long, you can remove Sec because you here use Duration, not Integer Second.

Reply:
done